libmscore/style.cpp

```diff
@@ -834,7 +834,8 @@ static const StyleType styleTypes[] {
       { Sid::measureNumberFontSpatiumDependent, "measureNumberFontSpatiumDependent", true },
       { Sid::measureNumberFontStyle,        "measureNumberFontStyle",       int(FontStyle::Normal) },
       { Sid::measureNumberColor,            "measureNumberColor",           QColor(0, 0, 0, 255) },
-      { Sid::measureNumberOffset,           "measureNumberOffset",          QPointF(0.0, 0.0) },
+      { Sid::measureNumberPosAbove,         "measureNumberOffset",          QPointF(0.0, -2.0) }, // This measureNumberOffset cannot be renamed to measureNumberPosAbove for backward compatibility
```

Comment:
FWIW, you probably *could* rename it and add code when reading that tag to actually set the other, and same on writing.  But, this is certainly the simpler solution.

Reply:
To me not worth the trouble of doing it differently than this.